src/components/ArrowKeyFocusManager.js

```diff
@@ -119,7 +123,20 @@ class ArrowKeyFocusManager extends Component {
     }
 }
 
-ArrowKeyFocusManager.propTypes = propTypes;
-ArrowKeyFocusManager.defaultProps = defaultProps;
+function ArrowKeyFocusManager(props) {
+    const isFocused = useIsFocused();
+
+    return (
+        <BaseArrowKeyFocusManager
+            // eslint-disable-next-line react/jsx-props-no-spreading
+            {...props}
+            isFocused={isFocused}
+        />
+    );
+}
+
+BaseArrowKeyFocusManager.propTypes = propTypes;
+BaseArrowKeyFocusManager.defaultProps = defaultProps;
+ArrowKeyFocusManager.displayName = 'ArrowKeyFocusManager';
```

Comment:
@ZhenjaHorbach We should create a folder with the name ArrowKeyFocusManager, and under this folder, BaseArrowKeyFocusManager.js and index.js files.
```javascript
ArrowKeyFocusManager
    BaseArrowKeyFocusManager.js
    index.js
```

Reply:
I was searching for GH issues and wondering if is there any ticket was created to replace ArrowKeyFocusManager with useArrowKeyFocusManager usage For SelectionList

Reply:
I saw this [comment](https://github.com/Expensify/App/issues/24985#issuecomment-1842637898) today, and If ArrowKeyFocusManager is going to be removed, then we should not add any logic to the ArrowKeyFocusManager.

Reply:
@ZhenjaHorbach Opened slack discussion https://expensify.slack.com/archives/C01GTK53T8Q/p1712043312366359 for confirming if we're going to replace the ArrowKeyFocusManager with useArrowKeyFocusManager for SelectionList, if there is no plan in near future, we can go with alternate proposal 

cc: @pecanoro

Reply:
For the BaseSelectionList Component, in this [PR](https://github.com/Expensify/App/pull/39201), ArrowKeyFocusManager is being replaced with useArrowKeyFocusManager, and it looks like in a [day or 2,](https://expensify.slack.com/archives/C01GTK53T8Q/p1712052024753809?thread_ts=1712043312.366359&cid=C01GTK53T8Q) PR will be merged; I think we can hold current PR until then other PR is merged, and once ArrowKeyFocusManager will be removed from the BaseSelectionList, we can retest [this issue](https://github.com/Expensify/App/issues/39093) again, and if it's not fixed by the useArrowKeyFocusManager update, we can do existing PR changes in useArrowKeyFocusManager hook.

cc: @ZhenjaHorbach @pecanoro

Reply:
Hmm, based on the last comments on that PR, I feel it's going to take at least a week since the tests are failing and there were a few bugs, maybe we should move forward with this one. Or are you afraid that the PR will undo the changes we are doing here?

Reply:
Then I would say, let's merge this one

Reply:
Then let's move forward with this PR 👍

Reply:
Let's finish reviewing this PR and I will post in the other issue to let them know about the changes here. But at this point I am not sure which one is closer to get merged 😄

Reply:
Then I would say I'll complete the checklist in the next hour 😃